src/main/java/frc/robot/Constants.java

```diff
@@ -273,7 +273,7 @@ public static class LEDConstants {
     public static final int DEVICE_ID = 9;
 
     public static final Color NOTE_COLOR = Color.fromHSV(3, 255, 255);
-
+    public static final Color FLASH_COLOR = Color.fromHSV(255, 255, 255);
```

Comment:
Why is the LED color different when doing baby-bird in this mode?

Reply:
To signify to the human player that the robot is baby-birding and needs a note

Reply:
Blinking with `NOTE_COLOR` already signifies that the robot is in baby-bird mode and the human player should load a note. Why is `FLASH_COLOR` a different color?
I'm not understanding why we flash orange for the human player when baby-birding by itself, but blinking white when baby-birding to stockpile.

Reply:
We definitely could, Osowski and Scott told me that we'd want the brightest we can achieve to notify the operator that the robot needs the note. There's no issue with changing it to orange though.

Reply:
I'm OK changing the color, my question is why is it _different_ from regular baby bird. If you want to use white, that's fine but give the constant a meaningful name (like `BABY_BIRD_COLOR`) and use it for _both_ ways the driver can trigger baby bird. You left the old one using `NOTE_COLOR`.